Scripts/Core/EditorVR.cs

```diff
@@ -96,6 +96,7 @@ static void ResetPreferences()
 
 		void Awake()
 		{
+			ObjectUtils.hideFlags = defaultHideFlags;
```

Comment:
Exactly. One would expect this to take effect the next time I restart EVR (or immediately, while EVR is running, but that would be going above and beyond).

I had considered leaving the one in the static constructor as well, but that seemed redundant.

I put break points on all the places where the value is used, and none are used before Awake. This may not always be the case, of course. Are we in favor of putting it back into the static ctor as well as leaving it here?

Also, @amirebrahimi would you like to continue to be added to PRs on core changes?